### PR TITLE
Keywords on P

### PR DESCRIPTION
**Pull request by joakim-hove**
Backwards

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
image

With this PR we have actually added the Json description of the last[1] Eclipse100 keyword. Thank you to @stefoss23 for a great effort.

[1]: Of course - there is always a disclaimer; there are roughly 10 keywords which have not been added, because they do not fit into the current C++ scheme. Hopefully not to hard to fix.

**Comment by atgeirr** [comment]
> With this PR we have actually added the Json description of the last[1] Eclipse100 keyword. Thank you to @stefoss23 for a great effort.

All I have to say is: wow! How many keywords are there now? (Keyword in the sense that it's got a JSON description.)

**Comment by joakim-hove** [comment]
> All I have to say is: wow! How many keywords are there now? (Keyword in the sense that it's got a JSON description.)

There are roughly 1050 json files - most of them have only one keyword, but e.g. the ones covering summary data have many - quite cool anyway!

File: src/opm/parser/eclipse/share/keywords/000_Eclipse100/P/PLYROCKM [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{"name" : "PLYROCK" , "sections" : ["SCHEDULE"], "size" : {"keyword":"TABDIMS" , "item":"NTSFUN"}, "items":
+    [   {"name":"IPV", "value_type" : "DOUBLE", "dimension":"1" },
```
Comment (totto82) [inline src/opm/parser/eclipse/share/keywords/000_Eclipse100/P/PLYROCKM]: @joakim-hove  Why is PLYROCK added to the SCHEDULE section and what happens when both PLYROCKM and PLYROCK has name PLYROCK?

Comment (joakim-hove) [reply src/opm/parser/eclipse/share/keywords/000_Eclipse100/P/PLYROCKM]: The `PLYROCKM`keyword should actually be in the SCHEDULE section, however the bug here is that there is a misatch between the json file `PLYROCKM` and the internal keyword name `PLYROCK` - a PR updating the latter to `PLYROCK` can be found here: https://github.com/OPM/opm-common/pull/1268

**Pull request merged**